starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+
+const StyledName = styled.h1`
+font-size: 50px;
+color: red;
+`
+
+const StyledContainer = styled.div`
+border: 2px solid black;
+margin: 2rem 0 2rem 0;
+`
+
+
+
+
+function CharacterList(props) {
+    return (
+        <div>
+            {props.people.map(swPeople => (
```

Comment:
`swPeople` seems like a bad name for this variable.
`people` is an array of characters. So when mapping through I would use `person` which is the singular of `people`